libs/core/src/lib/icon-tab-bar/types/icon-tab-bar-items.interface.ts

```diff
@@ -0,0 +1,27 @@
+import { SemanticColor } from './index';
```

Comment:
Interesting that we are going for a config style approach here rather than a more verbose template based approach like we did with the core Tab component.  Did I miss a discussion on this?

Reply:
Actually, before the implementation, I had doubts about which approach to take (even without knowing about the existence of a similar component).
But I chose the implementation with the config, because in my opinion this is a top level component and cannot be the basis for some other higher level wrapper components.
I followed these requirements from the [guideline](https://experience.sap.com/fiori-design-web/icontabbar/) and the API example from the [UI5](https://sapui5.hana.ondemand.com/#/api/sap.m.IconTabBar)

I tried to make this component a flexible and with user-friendly API.